modules/programs/sagemath.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.programs.sagemath;
+
+  inherit (lib) literalExample mkEnableOption mkIf mkOption types;
+
+  # Upstream does not use XDG by default :(.
```

Comment:
We should make it use XDG by default. I think the best way is to just change the default for the `configDir` to the XDG dir.